helm/hpcc/templates/_helpers.tpl

```diff
@@ -891,8 +865,8 @@ Pass in dict with root and me
 {{- define "hpcc.addSashaVolumeMounts" }}
 {{- $serviceName := printf "sasha-%s" .me.name -}}
 {{- if .me.storage }}
-{{- $volumeName := (hasKey .me.storage "plane") | ternary (printf "%s-pv" .me.storage.plane) $serviceName }}
-{{ include "hpcc.addVolumeMount" (dict "root" .root "me" .me.storage "name" $volumeName) -}}
+{{- $sashaStoragePlane := .me.storage.plane | default (include "hpcc.getFirstPlaneForCategory" (dict "root" .root "category" "sasha")) }}
```

Comment:
as per last comment, I think it would be better to get rid of storageType in schema for sasha at this juncture and just have option of plane.

Reply:
So
`.me.plane
`or
`.me.sashaPlane?
`